pkg/cloudprovider/providers/aws/aws.go

```diff
@@ -2790,9 +2790,16 @@ func (c *Cloud) EnsureLoadBalancer(clusterName string, apiService *v1.Service, n
 	return status, nil
 }
 
+// GetLoadBalancerName returns the name of the load balancer based on the service UID.
+func (c *Cloud) GetLoadBalancerName(service *v1.Service) string {
+	ret := string(service.UID)
```

Comment:
No @thockin , I have not checked API docs for cloud providers. So I just used old DeprecatedGetLoadBalancerName() for all the cloud providers other than GCE and fake. Can you point me to their API docs or is there  any aws(or any other cloudprovider) guy who would be able to help with this?